requirements.txt

```diff
@@ -31,3 +30,5 @@ mkdocs-material
 
 # py3.10 workarounds
 https://github.com/aaugustin/websockets/archive/4e1dac362a3639b9cf0e5bcf382601e6e32cfede.tar.gz#egg=websockets; python_version >= '3.10'
+
+anyio @ git+https://github.com/graingert/anyio.git@b907c72ffae4e89c09fc9d7d81ff8c45fee43ccd
```

Comment:
Why adding this, then?

Reply:
This is investigating https://github.com/agronholm/anyio/pull/358 it won't be needed if python 3.6 is dropped

Reply:
Pytest was dropped. Removing this.